### PR TITLE
Add metrics columns to teacher_feedbacks table

### PR DESCRIPTION
**Pull request by maddiedierker**
Initial PR for [LP-82](https://codedotorg.atlassian.net/browse/LP-82).

[Spec](https://docs.google.com/document/d/14Q_Eqe_allOhLFZN6P9IRXLU461zTm_9NAHLjoY7UR0/edit?usp=sharing)

Add new columns to `teacher_feedbacks` to use for gathering the following metrics:
1. `student_visit_count` - counter of student visits after feedback was left
2. `student_initial_visited_at` - timestamp of the first visit after feedback was left
3. `student_most_recent_visited_at` - timestamp of most recent visit after feedback was left

### CODE REVIEW

File: dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddMetricsToUserFeedbacks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :teacher_feedbacks, :student_visit_count, :integer
+    add_column :teacher_feedbacks, :student_initial_visited_at, :timestamp
+    add_column :teacher_feedbacks, :student_most_recent_visited_at, :timestamp
```
Comment (davidsbailey) [inline dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb]: sorry to nit, but what do you think about calling these `student_first_visited_at` and `student_last_visited_at`? `[first|last]_xxx_at` follow a pattern used elsewhere in our codebase and read more clearly (in my opinion). I know these are kind of annoying to change in db migrations, but I think we'll be happy later for these to have great names!

Comment (maddiedierker) [reply dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb]: that's quite alright, sounds good to me!

Comment (davidsbailey) [reply dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb]: If you don't like those, I think `student_initially_visited_at` and `student_most_recently_visited_at` would read a bit more easily and would be closer to what you originally had.

Comment (maddiedierker) [reply dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb]: i think following the first/last convention makes sense. naming is hard 😆

**Review by davidsbailey — APPROVED** [review]
thank you!

File: dashboard/db/schema.rb [hunk 2]
```diff
@@ -1372,14 +1372,17 @@
   end
 
   create_table "teacher_feedbacks", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
-    t.text     "comment",     limit: 65535
+    t.text     "comment",                  limit: 65535
```
Comment (dmcavoy) [inline dashboard/db/schema.rb:1374]: Should the extra space added here be worrisome?

**Review by dmcavoy — APPROVED** [review]
Looks good - Just one thing that is more of a question.

File: dashboard/db/schema.rb (hunk 2 above)
Comment (maddiedierker) [reply dashboard/db/schema.rb:1374]: nope, when you add/remove columns on the schema, the spacing is updated so that everything aligns

**Pull request merged**